src/indra_cogex/apps/wsgi.py

```diff
@@ -46,6 +47,10 @@
 app.extensions[INDRA_COGEX_EXTENSION] = Neo4jClient()
 app.extensions[STATEMENT_CURATION_CACHE] = CurationCache()
 
+neo4j_client = Neo4jClient()
+agent_cache = neo4j_client.load_agent_cache()
```

Comment:
Instead of running this at each startup, we could have a file-based cache and add its creation to the Dockerfile at build time.